backend/backend/urls.py

```diff
@@ -2,89 +2,106 @@
+from django.conf.urls.static import static
 
 from pong.consumers import PongConsumer
 
-# add app views here
-from User.views import UserDetailView, \
-						UserListView, \
-						UserProfilePictureView, \
-						UserLoginView, \
-						UserLogoutView, \
-						UserAdminDetailsView
-
-from Friends.views import FriendsListView, \
-						FriendshipDetailView
-
-
+# User views
+from User.views import (
+    UserDetailView,
+    UserListView,
+    UserProfilePictureView,
+    UserLoginView,
+    UserLogoutView,
+    UserAdminDetailsView
+)
+
+# Friends views
+from Friends.views import (
+    FriendsListView,
+    FriendshipDetailView
+)
+
+# Tokens views
 from Tokens.views import SingleMatchGuestTokenView
 
-from Match.views import MatchView, \
-							LaunchTestMatchView
-
-from Tournament.views import TournamentListView, \
-								TournamentDetailView, \
-								TournamentPlayerListView, \
-								TournamentPlayerDetailView, \
-								TournamentMatchListView, \
-								TournamentMatchDetailView
-
-from stats.views import StatsView, \
-							LeaderBoardView
-
+# Match views
+from Match.views import (
+    MatchView,
+    LaunchTestMatchView
+)
+
+# Tournament views
+from Tournament.views import (
+    TournamentListView,
+    TournamentDetailView,
+    TournamentPlayerListView,
+    TournamentPlayerDetailView,
+    TournamentMatchListView,
+    TournamentMatchDetailView
+)
+
+# Stats views
+from stats.views import (
+    StatsView,
+    LeaderBoardView
+)
 
 urlpatterns = [
-	path('admin/', admin.site.urls),
-
-	# User views
-	path('users/', UserListView.as_view()),
-	path('users/<int:user_id>', UserDetailView.as_view()),
-	path('users/<int:user_id>/profile_pictures/', UserProfilePictureView.as_view()),
-	path('login/', UserLoginView.as_view()),
-	path('logout/', UserLogoutView.as_view()),
-	path('admins/', UserAdminDetailsView.as_view()),
-
-	# Friends views
-	path('friends/', FriendsListView.as_view()),
-	path('friends/<int:friend_id>', FriendshipDetailView.as_view()),
-
-	# Tokens views
-	path('tokens/match/', SingleMatchGuestTokenView.as_view()),
-
-	# Match Views
-	path('match/', MatchView.as_view()),
-	path('match/test/', LaunchTestMatchView.as_view()), # TEMPORARY
-
-	# Tournament views
-	path('tournaments/', TournamentListView.as_view()),
-	path('tournaments/<int:tournament_id>', TournamentDetailView.as_view()),
-	path('tournaments/<int:tournament_id>/players/', TournamentPlayerListView.as_view()),
-	path('tournaments/<int:tournament_id>/players/<int:tournamentplayer_id>', TournamentPlayerDetailView.as_view()),
-	path('tournaments/<int:tournament_id>/matches/', TournamentMatchListView.as_view()),
-	path('tournaments/<int:tournament_id>/matches/<int:tournament_match_id>', TournamentMatchDetailView.as_view()),
-
-	# Stats Views
-	path('stats/', StatsView.as_view()),
-	path('leaderboard/', LeaderBoardView.as_view()),
+    path('admin/', admin.site.urls),
+
+    # User views
+    path('users/', UserListView.as_view(), name='user-list'),
```

Comment:
the name=user-list would be for use IF we used templates but as we dont, Im not sure we should add them here. Checking the documentation might help. But if it doesnt break anything than fine..

Reply:
just linking this issue here as itll be done in this one too, nice! https://github.com/dorianjagusch/ft_transcendence/issues/237

Reply:
Will deal with it. Name is indeed not necessary

Reply:
I removed the name stuff!